docs/BLUE_GREEN_MIGRATION.md

```diff
@@ -1,45 +1,86 @@
-	b. If `destination-table-version` and `source-table-version` do not exist in the deploy table, create destination-table-version record using this command:
+2. If `destination-table-version` and `source-table-version` do not exist in the deploy table, create destination-table-version record using this command:
 	```aws dynamodb put-item --region us-east-1 --table-name "efcms-deploy-${ENV}" --item '{"pk":{"S":"destination-table-version"},"sk":{"S":"destination-table-version"},"current":{"S":"1"}}'```
-	c. If `destination-table-version` and `source-table-version` do exist in the deploy table, increment destination-table-version by 1.
-3) If a new dynamo table and elasticsearch domain is necessary, duplicate the modules found in `web-api/terraform/template/main.tf`. Be sure to update the version number at the end of the module names, `table_name`, and `domain_name`. Do not delete the old modules. Add the modules to `depends_on` in `main-east.tf` and `main-west.tf`
-4) Run a deploy in circle.
-5) Verify the new application works at: 
+3. If `destination-table-version` and `source-table-version` do exist in the deploy table, increment destination-table-version by 1.
+
+## If a Migration is not necessary
+
+Still figuring this out and testing
+
+## If a new DynamoDB table and Elasticsearch domain is necessary
+
+If a new dynamo table and elasticsearch domain is necessary, duplicate the modules found in `web-api/terraform/template/main.tf`. Be sure to update the version number at the end of the module names, `table_name`, and `domain_name`. Do not delete the old modules. Add the modules to `depends_on` in `main-east.tf` and `main-west.tf`
+
+1. Run a deploy in circle.
+2. Verify the new application works at: 
 	- https://<DEPLOYED_COLOR>-<ENV>.<ZONE_NAME>
 	- https://app-<DEPLOYED_COLOR>.<ENV>.<ZONE_NAME>
-6) Destroy the migration infrastructure to turn off the live streams
+3. Destroy the migration infrastructure to turn off the live streams
 	`DESTINATION_TABLE=b SOURCE_TABLE=a STREAM_ARN=abc npm run destroy:migration -- <DEPLOYED_ENV>`
 
 ## Errors You May Encounter
+
 ### AWS-related
+
 #### Route53 Error
+
 ```
 Error: [ERR]: Error building changeset: InvalidChangeBatch: [Tried to create resource record set [name='_243f260ea635a6dffe0db2c6cc1c1158.*************************.', type='CNAME'] but it already exists]
 ```
 Manually delete the Route53 record and rerun the deploy.
 
 
 #### IAM Role already exists
+
 ```
 Error: Error creating IAM Role migration_role_<ENV>: EntityAlreadyExists: Role with name 		migration_role_<ENV> already exists.
 	status code: 409, request id: ***********
 
   on migration.tf line 1, in resource "aws_iam_role" "migration_role":
    1: resource "aws_iam_role" "migration_role" {
-   ```
-Delete the role in the AWS IAM console and rerun `npm run deploy:environment-specific <ENV>`.
+```
 
+AWS IAM console and rerun `npm run deploy:environment-specific <ENV>`.
```

Comment:
Did you intend to omit `Delete the role` from the beginning of this line?

Reply:
oops, thanks!